### PR TITLE
Flight capships - Bretonia Battleship

### PR DESCRIPTION
**Pull request by FireFlyEx**
Todo:

Crash test - another player needs to crash into it to verify what happens.

Known issues (unavoidable??): because it turns so slowly it will not goto or dock with anything requiring a tight turn to line up with and it does not work well in formations.

### CODE REVIEW

File: DATA/SHIPS/shiparch.ini [hunk 1]
```diff
@@ -2357,7 +2357,7 @@ ids_info = 461098
 ids_info1 = 461099
 ;res html
 ; \m\bSmall Transport\B\l
-;
```
Comment (erikroe) [inline DATA/SHIPS/shiparch.ini:2360]: This and the following removed empty spaces break the infocard of the transport.

File: DATA/EQUIPMENT/engine_equip.ini [hunk 2]
```diff
@@ -3258,9 +3258,9 @@ ids_info = 460270
 ; Developed by the Government supported avionics supplier \iRoyal Avionics\I, this engine serves as a reliable asset to all heavy Bretonian armed forces.\n
 volume = 0
 mass = 20
-max_force = 8000000
+max_force = 24000000
 linear_drag = 1
-reverse_fraction = 0.5
+reverse_fraction = 5.5
```
Comment (erikroe) [inline DATA/EQUIPMENT/engine_equip.ini:3263]: Why should the ship go 550% of its forward speed in reverse?

**Comment by erikroe** [comment]
Please change all your commits to actual useful contents.
Ideally they begin with a verb like "Add" or "Change" and then saying what this commit does in a single sentence.

**Comment by erikroe** [comment]
Please create a Github Issue that defines the actual goals of what you do here.

**Review by erikroe — CHANGES_REQUESTED** [review]

**Comment by erikroe** [comment]
About the testing:
I tried ramming the battleship with another fighter. Same results as usual: It spins. Not terribly, but it does. Just the same as before your changes. No improvement or worsening.
About the todos: That's a classic problem. The slow turning speed is not the issue I think. It's rather the HpMount being very far away from the ship's center. And this confuses the autopilot. Because HpMount is relevant for docking, whereas the center of the ship is for turning. The autopilots were not made for such discrepancies.

**Comment by erikroe** [comment]
Generally I like the steering of the ship. It does not feel any different from the previous one really. Still sluggish, as it should be.

File: DATA/EQUIPMENT/engine_equip.ini (hunk 2 above)
Comment (FireFlyEx) [reply DATA/EQUIPMENT/engine_equip.ini:3263]: Confirmed and changed.

File: DATA/SHIPS/shiparch.ini (hunk 1 above)
Comment (FireFlyEx) [reply DATA/SHIPS/shiparch.ini:2360]: All of these have been fixed now.

**Pull request closed without merging**